src/components/Textarea/Textarea.js

```diff
@@ -26,19 +29,31 @@ const Textarea = ({
   id,
   value,
   onChange,
+  onBlur,
   variant,
   size,
   i18n,
   hasResize,
   maxHeight,
-  disabled
+  disabled,
+  errors,
+  touched
 }) => {
+  const error = errors[name]
+  const isTouched = touched[name]
```

Comment:
Kuknij do /form/input na helper getDeepValue dodałbym to tutaj dla spójności
Pomaga to gdy formik ma zagnieżdżony stan `np const errrors = {object: {textarea: true}}` aby wydobywać errora z zagnieżdżonego obiektu

Reply:
rozwiązanie jak najbardziej poprawne, ale nie zadziała jak formularz będzie bardziehj rozbudowany :P